src/plays/inspirational-quotes/card.css

```diff
@@ -0,0 +1,72 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;0,500;0,700;1,400&display=swap");
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+}
+.card {
```

Comment:
Same comment for all the styles..

.inpirational-quotes-play .card

for example.

Reply:
@muzaffar640 This comment is pending.